### DISCUSSION TITLE
Some ideas about the Tools page UI

### DISCUSSION THREAD

**Discussion by CloneWith** [kylecorry31/Trail-Sense#2205] [New Feature Ideas]
On bigger devices like tablets, there is more than enough space beside each tool's button. And for a menu like this:

![IMG_20240205_202709.png](https://github.com/kylecorry31/Trail-Sense/assets/110881926/c14f29c4-e438-498c-9eba-504a9346e231)

- The `experimental` item isn't actually used, so using a badge on the icon to clarify it might be a good idea.
- Some tools have an `About` item, which can be placed just *under* the icon.
- For `User Guide`, there can be a more accessible place (e.g. the right side of the button).

so hard to describe by text 😓 posting a screenshot of Image Toolbox for reference:

![Screenshot_20240205-204158.png](https://github.com/kylecorry31/Trail-Sense/assets/110881926/bed0aa21-0d82-4784-bd53-2b4f052f90fb)

---

btw how about placing `Sensor` and `User Guide` buttons at the top bar? This may make them easier to find and use.

**Comment by kylecorry31** [comment]
@CloneWith thank you for the suggestions:

1. On your device do you see 2 columns for the tool buttons and they are still that large (I'm assuming it is a really big screen)? I haven't really considered tablet form factors for Trail Sense, as they normally aren't used while hiking. But I can experiment with a 3 column layout on  large devices.
2. I looked into the badge for that, but decided that since experimental is always opt-in and temporary that it wasn't worth the extra layout complexity.
3. I previously had the about text on the tools layout, but removed it since it took up a lot of space (some descriptions may be long) and wasn't very useful once you've used the tools.
4. Similar to 3, on small sized screens there isn't a lot of room, and while the user guide is useful, as people start using the tool they will need to refer to that guide less, meaning it takes up space and is unused. I can experiment with maybe having a menu button show on the tools to make it clear there are multiple options there.
5. On smaller sized screens the sensor and user guide buttons would take up too much room at the top (cutting into the searchbar). The sensor tool is not used often except for debugging (and by people curious about their device sensors - but not really for hiking), and I have the user guide as a pinned tool instead for new installs (so they can unpin it after they get used to Trail Sense)

**Reply by kylecorry31** [reply]
I've added the following issues to track the changes I plan on making:
- https://github.com/kylecorry31/Trail-Sense/issues/2206
- https://github.com/kylecorry31/Trail-Sense/issues/2207
